src/accel_mode_and_odr.rs

```diff
@@ -15,8 +20,14 @@ where
     /// Returns `Error::InvalidInputData` if the mode is incompatible with
     /// the given output data rate.
     ///
-    #[doc = include_str!("delay.md")]
-    pub fn set_accel_mode_and_odr<D: DelayNs>(
+    /// The given `delay` is used to wait for the sensor to turn on or change modes,
+    /// according to the times specified in Table 14 and Table 15 in the [datasheet].
+    /// You can opt out of this by using a no-op delay implementation, see
+    /// [`embedded_hal_mock::delay::MockNoop`] for an example of such an
+    /// implementation.
+    /// [datasheet]: https://www.st.com/resource/en/datasheet/lsm303agr.pdf
+    /// [`embedded_hal_mock::delay::MockNoop`]: https://docs.rs/embedded-hal-mock/latest/embedded_hal_mock/delay/struct.MockNoop.html
+    pub async fn set_accel_mode_and_odr<D: DelayNs>(
```

Comment:
I copied in the contents of `delay.md` here as well as in [magnetometer.rs:16-32](https://github.com/eldruin/lsm303agr-rs/pull/31/files#diff-655dfc5c14daec4123c6f7da6fd3026f88f91db687ed9cfc2a761bac52131ca6L16-R32)

Not too happy about the reduncancy, but the expansion of `maybe_async_cfg::maybe` does not work well with `#[doc = include_str!("<file>")]`

Reply:
ok no worries. Please delete the file since there are no users anymore.